README.md

```diff
@@ -1,5 +1,5 @@
 # react-math-magicians
-This is the first project of the Math Magicians application. 
+In this project, I continue with the development of the Math Magicians app. I develop a React component that will hold the core functionality: a calculator. 
```

Comment:
- [ ] please make sure to use[ the template](https://github.com/microverseinc/readme-template) for the readme file provided by Microverse, including the links (for example to the MIT license). keep in mind that all sections of the template are mandatory except for the "Live Demo", "Run Tests" and "Deployment" sections, these are only mandatory if the specific project you're working on requires it.

Reply:
Hi @mike, thanks for your comment.